website/project/views/register.py

```diff
@@ -40,25 +38,18 @@
 from website.identifiers.client import EzidClient
 
 from .node import _view_project
-from .. import clean_template_name
-
 
 @must_be_valid_project
 @must_have_permission(ADMIN)
-@must_not_be_registration
 def node_register_page(auth, node, **kwargs):
 
-    ret = {
-        'options': [
-            {
-                'template_name': metaschema['name'],
-                'template_name_clean': clean_template_name(metaschema['name'])
-            }
-            for metaschema in OSF_META_SCHEMAS
-        ]
-    }
-    ret.update(_view_project(node, auth, primary=True))
-    return ret
+    if node.is_registration:
```

Comment:
It took a while to find this view's function, but I did. Now I have a few questions about it:

* It's only valid for registrations. For any other node, it redirects the user to the node's registrations page with a message. I'd like to remove this redirect, but I want to try to understand why it was put there in the first place, first.
* If we get rid of the view for nodes, I'd like to use the `must_be_registration` decorator.
* The URL is misleading. It's attached at `/project/<pid>/register/`. I would prefer `/project/<pid>/registered_metadata/` or similar. Thoughts?